Algorithm.Framework/Selection/ManualUniverse.cs

```diff
@@ -60,18 +62,26 @@ public class ManualUniverse : UserDefinedUniverse
         /// <param name="subscriptionService">Instance which implements <see cref="ISubscriptionDataConfigService"/> interface</param>
         /// <returns>All subscriptions required by this security</returns>
         public override IEnumerable<SubscriptionRequest> GetSubscriptionRequests(Security security, DateTime currentTimeUtc, DateTime maximumEndTimeUtc,
-                                                                                ISubscriptionDataConfigService subscriptionService)
+            ISubscriptionDataConfigService subscriptionService)
         {
-            return security.Subscriptions.Select(config =>
-                new SubscriptionRequest(
-                    isUniverseSubscription: false,
-                    universe: this,
-                    security: security,
-                    configuration: new SubscriptionDataConfig(config, isInternalFeed: false),
-                    startTimeUtc: currentTimeUtc,
-                    endTimeUtc: maximumEndTimeUtc
-                )
-            );
+            // ManualUniverse will return any existing SDC for the symbol, else will create new, using universe settings.
+            // This is for maintaining existing behavior and preventing breaking changes: Specifically motivated
+            // by usages of Algorithm.Securities.Keys as constructor parameter of the ManualUniverseSelectionModel.
+            // Symbols at Algorithm.Securities.Keys added by Addxxx() calls will already be added by the UserDefinedUniverse.
```

Comment:
Thank you for adding this note. Future generations (or maybe future us) will thank you greatly :)